src/main/java/uk/gov/hmcts/darts/task/service/impl/AdminAutomatedTasksServiceImpl.java

```diff
@@ -103,6 +111,48 @@ public DetailedAutomatedTask updateAutomatedTask(Integer taskId, AutomatedTaskPa
             log.info("Batch size for {} updated to {}", automatedTask.getTaskName(), automatedTaskPatch.getBatchSize());
         }
 
+        //Arm Autoamted Task updates
+        List<Consumer<ArmAutomatedTaskEntity>> armAutomatedTaskEntityConsumer = new ArrayList<>();
+
+        if (automatedTaskPatch.getArmReplayStartTs() != null) {
+            armAutomatedTaskEntityConsumer.add(armAutomatedTaskEntity -> {
+                armAutomatedTaskEntity.setArmReplayStartTs(automatedTaskPatch.getArmReplayStartTs());
+                log.info("ARM replay start timestamp for {} updated to {}", automatedTask.getTaskName(), automatedTaskPatch.getArmReplayStartTs());
```

Comment:
Is it possible that the `ArmReplayStartTs` will be populated with the existing value? If so, this log statement wouldn't really apply.